package.json

```diff
@@ -144,6 +145,7 @@
   },
   "peerDependencies": {
     "@bigbinary/neeto-commons-frontend": "latest",
+    "@bigbinary/neeto-molecules": "latest",
```

Comment:
We should not add it to peer dependency before it added to `package-common.json` file. We'll make this change once that is done.